docs/designers-developers/developers/tutorials/block-based-themes/README.md

```diff
@@ -16,7 +16,7 @@ Additional documentation for block-based themes is available at https://develope
  2. [Creating the theme](/docs/designers-developers/developers/tutorials/block-based-themes/README.md#creating-the-theme)
  3. [Creating the templates and template parts](/docs/designers-developers/developers/tutorials/block-based-themes/README.md#creating-the-templates-and-template-parts)
  4. [Experimental-theme.json -Global styles](/docs/designers-developers/developers/tutorials/block-based-themes/README.md##experimental-themejson--global-styles)
- 5. [Adding blocks](/docs/designers-developers/developers/tutorials/block-based-themes/adding-blocks.md)
+ 5. [Adding blocks](docs/designers-developers/developers/tutorials/block-based-themes/block-based-themes-2-adding-blocks.md)
```

Comment:
You do want the slash at the beginning, see [the Using Links in docs guide](https://developer.wordpress.org/block-editor/contributors/document/#using-links)

```suggestion
 5. [Adding blocks](/docs/designers-developers/developers/tutorials/block-based-themes/block-based-themes-2-adding-blocks.md)
```